rflx/model/message.py

```diff
@@ -1072,20 +1085,19 @@ def __compute_topological_sorting(self) -> Optional[Tuple[Field, ...]]:
             return None
         return tuple(f for f in result if f not in [INITIAL, FINAL])
 
-    def __compute_paths(self, final: Field) -> Set[Tuple[Link, ...]]:
-        if final == INITIAL:
-            return {()}
-        return set(
-            itertools.chain.from_iterable(
-                ((p + (l,) for p in self.__compute_paths(l.source)) for l in self.incoming(final))
-            )
-        )
+    def __compute_reverse(self) -> Dict[Field, List[Link]]:
+        result: Dict[Field, List[Link]] = {
+            f: [] for l in self.structure for f in (l.source, l.target)
```

Comment:
Isn't `for l in self.structure for f in (l.source, l.target)` the same as `for f in self.all_fields`?

Reply:
Yes, but I removed the method all together. See below.